### PR TITLE
Single Video Page [WIP] 👨‍🚀

### PR DESCRIPTION
**Pull request by sudovijay**
Continue From PR - #91

### CODE REVIEW

**Comment by sudovijay** [comment]
lol @SaraVieira  i was telling ya this slug thing ain't gonna work, i mean until we don't store them separately it just not going to work.. there is just too much stuff to be taken care of .

Earlier @cbh6  was using `name` in gql and now he moved to `name_contains` just to solve that video error. but it ends up with more issues, have a look to these urls

- https://saravieira-awesome-talks-video-page.now.sh/video/is-javascript-making-us-better-professionals
- https://saravieira-awesome-talks-video-page.now.sh/video/is-javascript-making-us-
- https://saravieira-awesome-talks-video-page.now.sh/video/is-javascrip
- https://saravieira-awesome-talks-video-page.now.sh/video/Is%20JavaScript%20Making%20Us%20Better%20Professionals?
- https://saravieira-awesome-talks-video-page.now.sh/video/is-javascript
- https://saravieira-awesome-talks-video-page.now.sh/video/is
- etc ...

thats **contains** for ya

The problem with this URL is `?` , basically a special character, you can't have that in slug and so it can't be matched with name either . the name can actually have any special character so we have to take that in consideration too.

There are actually three ways to solve this 

1.  Either form slug when you publish video and store it, that way it'll be remain same everywhere
2. The other way is to remove all special characters from video name, that way we could match the slug and name 
3. Last option will be , use `Youtube ID ` in slug and match that, we could even have that encrypted a bit so it'll not look exactly like youtube id and get decrypt while we receive it lol

onto ya now 🙂

**Comment by SaraVieira** [comment]
That is a good point, we can use the id , I'll make that change real fast :)


It's simpler, otherwise I need to make another slug bot ahah

**Comment by SaraVieira** [comment]
Pushed and now it's by id in GraphCMS :)

**Comment by sudovijay** [comment]
your lazy and fast at the same time lol 🙂

**Comment by SaraVieira** [comment]
It's a skill man ahah

**Comment by sudovijay** [comment]
saw your tweet. it wasn't necessary, mentioning thing, so no thank you for that 🙂 

and i was thinking we should not shut this cinema thing off, i kinda like it, (Who doesn't like cinema?) what we could do is let user decide, if he clicks on name it takes him to the video page, while clicking on thumb will trigger cinema and click on play will start inline player ...

Even we could add a share box in both cinema & video page, that way user can easily copy & share the link ..

**Comment by SaraVieira** [comment]
Damn it ! ahah

Isn't that what happens now ?


How do you feel about this: https://saravieira-awesome-talks-video-page.now.sh/

**Comment by cbh6** [comment]
Hey guys ! That's getting very cool 👍

**Comment by sudovijay** [comment]
nope cinema is gone 🙁 anyway it looks good,  just the duration look bit off. idk , we can move that in right just in align with tags may be. what you think?

**Comment by SaraVieira** [comment]
Waaaat ? We should have cinema mode still :(


I also need to fix the speakers so yap, that makes sense :)

**Comment by cbh6** [comment]
Disabled cinemamode thinking you don't want it since there is a single video page. My bad. Hope it will get cool now @sudovijay enabled it again !

**Comment by sudovijay** [comment]
Nothing can beat cinema 🎦

**Comment by SaraVieira** [comment]
@sudovijay

File: src/Components/Favorite/view.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ import Heart from '../Styling/Heart'
 
 import randomID from 'random-id'
 
-export default ({ favorites, removeFavorite, addFavorite, id }) => {
+export default ({ favorites, removeFavorite, addFavorite, id, big }) => {
     const inputId = randomID()
     return (
-        <Heart>
+        <Heart big={big}>
```
Comment (pksjce) [inline src/Components/Favorite/view.js]: Such a big <3

**Comment by pksjce** [comment]
Wow! This looks so good!

**Comment by SaraVieira** [comment]
I'll try to give it some final touches, update the bot and merge it today :D

**Pull request merged**